src/huntsman/pocs/states/huntsman/ready.py

```diff
@@ -1,11 +1,16 @@
+""" Once in the "ready" state, Huntsman has been initialized successfully and it is safe. The goal
+of the ready state is to decide which of the following states to enter next:
+- parking
+- coarse_focusing
+- scheduling
+- twilight_flat_fielding
+"""
+
+
 def on_enter(event_data):
     """
-    Once in the `ready` state our unit has been initialized successfully. We now
-    decide on the next state and ready the cameras if appropriate.
     """
     pocs = event_data.model
-    pocs.next_state = 'parking'
```

Comment:
By removing this is there still a fail-safe for if something goes wrong below?

Reply:
See comment above. I think fail-safe should be handled by the state machine.